app/src/main/res/values/strings.xml

```diff
@@ -10,7 +10,8 @@
     <string name="not_connected">No Internet Connection</string>
     <string name="json_error">JSON Error</string>
     <string name="no_trackers">The application could contain tracker(s) we do not know yet.</string>
-    <string name="tested">There\'s no report for the installed version (%1$s), the information displayed is based on another (%2$s) version</string>
+    <string name="version_equals" translatable="false">%1$s = %2$s</string>
+    <string name="version_mismatch" translatable="false">%1$s &lt;&gt; %2$s</string>
```

Comment:
Replace this string by `Installed Version <> Analyzed Version`

Reply:
Done 6b6b0b97c6a27696e7335800b1106e1aed77b138